app/src/test/kotlin/co/netguru/baby/monitor/client/feature/client/home/SendBabyNameUseCaseTest.kt

```diff
@@ -23,13 +21,10 @@ class SendBabyNameUseCaseTest {
         on { getChildData() }.doReturn(Maybe.just(childDataEntity))
     }
     private val message = "message"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/b1aa35793dca6d7714e5534d6167592f0bbe6a99/app/src/test/kotlin/co/netguru/baby/monitor/client/feature/client/home/SendBabyNameUseCaseTest.kt#L23"></span>Private property message is unused.</td>
    </tr>
  </tbody>
</table>